packages/next/src/client/components/react-dev-overlay/_experimental/internal/container/runtime-error/component-stack-pseudo-html.tsx

```diff
@@ -318,90 +342,73 @@ export function PseudoHtmlDiff({
-    >
-      <path
-        fillRule="evenodd"
-        clipRule="evenodd"
-        d="M7.25 8.75V12H8.75V8.75H12V7.25H8.75V4H7.25V7.25H4V8.75H7.25Z"
-        fill="currentColor"
-      />
-    </svg>
-  )
-}
-
-function MinusIcon() {
-  return (
-    <svg
-      width="16"
-      height="16"
-      viewBox="0 0 16 16"
-      fill="none"
-      xmlns="http://www.w3.org/2000/svg"
-      className="error-overlay-hydration-error-diff-minus-icon"
-    >
-      <path
-        fillRule="evenodd"
-        clipRule="evenodd"
-        d="M12 8.75H4V7.25H12V8.75Z"
-        fill="currentColor"
-      />
-    </svg>
-  )
-}
-
 export const PSEUDO_HTML_DIFF_STYLES = css`
   [data-nextjs-container-errors-pseudo-html] {
-    border-top: 1px solid var(--color-gray-400);
+    padding: var(--size-3) 0;
+    margin: var(--size-2) var(--size-4) var(--size-4);
+    border: 1px solid var(--color-gray-400);
     background: var(--color-background-200);
     color: var(--color-syntax-constant);
     font-family: var(--font-stack-monospace);
     font-size: var(--size-font-smaller);
     line-height: var(--size-4);
+    border-radius: var(--size-2);
+  }
+  [data-nextjs-container-errors-pseudo-html-line] {
+    display: inline-block;
+    width: 100%;
+    padding-left: var(--size-10);
+  }
+  [data-nextjs-container-errors-pseudo-html-line--error] {
+    background: var(--color-red-300);
+    font-weight: bold;
   }
   [data-nextjs-container-errors-pseudo-html-collapse] {
     all: unset;
-    padding: var(--size-2);
+    margin-left: var(--size-3);
     &:focus {
       outline: none;
     }
   }
-  [data-nextjs-container-errors-pseudo-html--diff] {
-    width: fit-content;
-  }
   [data-nextjs-container-errors-pseudo-html--diff='add'] {
     background: var(--color-green-300);
   }
+  [data-nextjs-container-errors-pseudo-html--diff='add']::before {
+    content: '+';
+    color: var(--color-green-900);
+    float: left;
+    width: 0;
+    margin-left: calc(var(--size-6) * -1);
+    margin-right: var(--size-8);
+  }
   [data-nextjs-container-errors-pseudo-html--diff='remove'] {
     background: var(--color-red-300);
   }
-  [data-nextjs-container-errors-pseudo-html--tag-error] {
-    background: var(--color-red-300);
-    font-weight: bold;
+  [data-nextjs-container-errors-pseudo-html--diff='remove']::before {
+    content: '-';
+    color: var(--color-red-900);
+    float: left;
+    width: 0;
+    margin-left: calc(var(--size-6) * -1);
+    margin-right: var(--size-8);
   }
-  /* hide but text are still accessible in DOM */
+  ${/* hide but text are still accessible in DOM */ ''}
   [data-nextjs-container-errors-pseudo-html--hint] {
     display: inline-block;
     font-size: 0;
+    height: 0;
   }
   [data-nextjs-container-errors-pseudo-html--tag-adjacent='false'] {
     color: var(--color-accents-1);
   }
-  [data-nextjs-container-errors-pseudo-html] > pre > code > span {
-    display: block;
-    padding: var(--size-1) var(--size-4);
```

Comment:
this feels a little tight... I think we need a little bit of top/bottom padding

PR:
![CleanShot 2025-02-03 at 20 46 10@2x](https://github.com/user-attachments/assets/e5dfba8f-59c7-4ee3-9b3f-a84130a6de1d)

Design:
![CleanShot 2025-02-03 at 20 47 08@2x](https://github.com/user-attachments/assets/48cc5a9f-6186-424f-886f-5856e1dd6493)